File: JAVA PROGRAMMING/Test76.java
// 76. PRINT NUMBERS FROM RANGE1 TO RANGE2 AND DISPLAY THOSE NUMBERS WHICH ARE MULTIPLES OF A GIVEN NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range1 Input: ");
        int range1 = in.nextInt();

        System.out.println("Enter Range2 Input: ");
        int range2 = in.nextInt();

        System.out.println("Enter Target: ");
        int target = in.nextInt();

        for(int i = range1; i <= range2; i++) {
            if(i % target == 0) {
                System.out.print(i+" ");
            }
        }
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test76.ts
// 76. PRINT NUMBERS FROM RANGE1 TO RANGE2 AND DISPLAY THOSE NUMBERS WHICH ARE MULTIPLES OF A GIVEN NUMBER
const Func = (range1: number, range2: number, target: number): void => {
    for(let i = range1; i <= range2; i++) {
        if(i % target === 0) {
            process.stdout.write(`${i} `);
        }
    }
    console.log();
}
Func(100, 200, 14);
Func(200, 400, 35);